tests/test_utils.py

```diff
@@ -25,12 +25,18 @@
 )
 
 CKPT_MODEL_PATHS = {
-    "small_test_ckpt_tune": "/tmp/test-artifacts/small-ckpt-tune-03082024.pt",
-    "small_test_ckpt_meta": "/tmp/test-artifacts/small-ckpt-meta-03082024.pt",
-    "small_test_ckpt_hf": "/tmp/test-artifacts/small-ckpt-hf-03082024.pt",
+    "LLAMA2_tune": "/tmp/test-artifacts/small-ckpt-tune-03082024.pt",
+    "LLAMA2_meta": "/tmp/test-artifacts/small-ckpt-meta-03082024.pt",
+    "LLAMA2_hf": "/tmp/test-artifacts/small-ckpt-hf-03082024.pt",
+    "LLAMA3_tune": "/tmp/test-artifacts/small-ckpt-tune-llama3-05052024.pt",
```

Comment:
nit: can we keep these lowercase? I know that checkpointer ModelType enum is all caps, but for everything else I'd prefer to keep things lowercase. (Same comment for tokenizer_paths, model_test_configs, etc.)